stablehlo/dialect/StablehloOps.td

```diff
@@ -1117,17 +1117,16 @@ def StableHLO_AfterAllOp : StableHLO_Op<"after_all", [Pure]> {
   let summary = "AfterAll operator";
 
   let description = [{
-    AfterAll takes a variadic number of tokens and produces a single token.
-    Tokens are primitive types which can be threaded between side-effecting
-    operations to enforce ordering. AfterAll can be used as a join of tokens
-    for ordering a operation after a set operations.
+    Ensures that the operations producing the `inputs` are ordered before any
+    operation that depend on `result`.
```

Comment:
Looks like you might've forgotten to update the description?

Reply:
Yup. fixed!